docs/contribution/40-how-to/5-support.mdx

```diff
@@ -2,11 +2,15 @@
 title: How to Get Support for Extension Development
 ---
 
-We use the [mittwald/contributor-support](https://github.com/mittwald/contributor-support) Github project to exchange information between mittwald and its contributors. You can send questions and feature requests to mittwald there, contact other contributors, and submit a new extension or edit an existing one via an issue. Please note that this repository (and all issues therein) is public.
+As soon as you are listed as a contributor, you get access to a [private GitHub repository](https://github.com/mittwald/contributor-support) where all contributors and mittwald exchange information.
+You can send questions and feature requests to mittwald there, contact other contributors, and submit a new extension or edit an existing one via an issue.
```

Comment:
This is outdated information; https://github.com/mittwald/contributor-support is already public.